src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
If we're moving to doing this, can we initialize `this.container = null;` in the `constructor` so the variables are clear.